sites/all/themes/wille/sass/component/_carousel.scss

```diff
@@ -29,22 +45,18 @@ $carousel-outer-padding: 0 30px;
   li {
     display: inline-block;
     flex: none;
-    margin-bottom: 1em;
-    margin-right: 0;
-    width: 200px;
-    max-width: 30vw;
-
-
-    @include media($phone) {
-      margin-right: 5px;
-    }
+    margin: 0;
```

Comment:
Hvad er sizing logikken for covers? Historisk set har Geist givet os 2 størrelser, mobil og alt andet, og vi har så freestyle't derfra.

Reply:
Som vi snakkede om, så burde dette være i orden - lad os kigge på det sammen når det kommer ud på test :)